ansible/group_vars/all/defaults.yaml

```diff
@@ -14,7 +14,7 @@ kubernetes_cni_version: "0.9.1"
 # The project release closely follows the Kubernetes release cycle, and uses a
 # nearly identical version scheme.
 # IMPORTANT When you update crictl_version, also update crictl_sha256.
-crictl_version: "1.26.0"
+crictl_version: "1.28.0"
```

Comment:
Following the comment above, shouldn't `crictl_sha256` also been updated below?

Reply:
added. the sha256 file is just sitting in my Downloads directory 🤣